swan-lake/learn-the-platform/run-in-the-cloud/code-to-cloud/code-to-cloud-deployment.md

```diff
@@ -194,7 +208,7 @@ COPY wso2-hello_docker-0.1.0.jar /home/ballerina
 EXPOSE  9090
 USER ballerina
 
-CMD java -Xdiag -cp "wso2-hello_docker-0.1.0.jar:jars/*" 'wso2/hello_docker/0/$_init' || cat ballerina-internal.log
+CMD java -Xdiag -cp "anjana-http2_test-0.1.0.jar:jars/*" 'anjana/http2_test/0/$_init'
```

Comment:
Is it ok to have usernames?

Reply:
this is due to folder structure being different in my test setup. I'll change this in another PR ;)

Reply:
Got it! No problem.